website/static/css/pages/project-page.css

```diff
@@ -104,11 +104,13 @@
 .editable-error-block {
     font-size: 16px;
     max-width: 100%;
-  margin: 5px 0 0;
-  width: auto;
-  white-space: normal;
+    margin: 5px 0 0;
+    width: auto;
+    white-space: normal;
 }
 
 #addPointer td {
     vertical-align: middle;
-}
+    word-break: break-all;
+    word-break: break-word; /* Non standard for webkit*/
```

Comment:
@sloria Yes, it is because break-word is not implemented in all browsers. But in our case, break-word is better. So I put it at the end.